frontend/www/js/omegaup/components/qualitynomination/List.vue

```diff
@@ -1,79 +1,113 @@
 <template>
-  <div class="panel panel-default">
-    <div class="panel-heading">
-      <h3 class="panel-title">
+  <div>
+    <div class="form-inline" v-if="!myView">
+      <omegaup-autocomplete
+        v-bind:init="el => typeahead.nominationTypeahead(el)"
+        v-model="query"
+        v-bind:placeholder="T.wordsKeyword"
+        name="query"
+        class="form-control"
+      ></omegaup-autocomplete>
+      <select name="column" class="form-control" v-model="selectColumn">
+        <option
+          v-for="(columnText, columnIndex) in columns"
+          v-bind:value="columnIndex"
+        >
+          {{ columnText }}</option
+        >
+      </select>
+      <button
+        class="btn btn-primary"
+        v-on:click.prevent="
+          $emit('goToPage', 1, showAll ? 'all' : 'open', query, selectColumn)
+        "
+      >
+        {{ T.wordsSearch }}
+      </button>
+      <br /><br /><br />
+    </div>
+    <div class="card">
+      <h3 class="card-header">
         {{
           UI.formatString(T.nominationsRangeHeader, {
             lowCount: (pages - 1) * length + 1,
             highCount: pages * length,
           })
         }}
       </h3>
-    </div>
-    <div class="panel-body">
-      <a v-if="isAdmin" href="/group/omegaup:quality-reviewer/edit/#members">
-        {{ T.addUsersToReviewerGroup }}
-      </a>
-      <div class="pull-right" v-if="!myView">
-        <label>
-          <input
-            type="checkbox"
-            v-model="showAll"
-            v-on:change="$emit('goToPage', 1, showAll ? 'all' : 'open')"
-          />
-          {{ T.qualityNominationShowAll }}
-        </label>
+      <div class="card-body">
+        <a v-if="isAdmin" href="/group/omegaup:quality-reviewer/edit/#members">
+          {{ T.addUsersToReviewerGroup }}
+        </a>
+        <div class="pull-right" v-if="!myView">
+          <label>
+            <input
+              type="checkbox"
+              v-model="showAll"
+              v-on:change="
+                $emit(
+                  'goToPage',
+                  1,
+                  showAll ? 'all' : 'open',
+                  query,
+                  selectColumn,
+                )
+              "
+            />
+            {{ T.qualityNominationShowAll }}
+          </label>
+        </div>
       </div>
+      <table class="table table-striped">
+        <thead>
+          <tr>
+            <th>{{ T.wordsAlias }}</th>
+            <th v-if="!myView">{{ T.wordsNominator }}</th>
+            <th>{{ T.wordsAuthor }}</th>
+            <th>{{ T.wordsSubmissionDate }}</th>
+            <th v-if="!myView">{{ T.wordsReason }}</th>
+            <th class="text-center">{{ T.wordsStatus }}</th>
+            <th><!-- view button --></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="nomination in nominations.nominations">
```

Comment:
Aquí con esto es suficiente:
```suggestion
          <tr v-for="nomination in nominations">
```

Reply:
listo